frontend/src/features/properties/map/research/common/ResearchHeader.tsx

```diff
@@ -11,54 +12,69 @@ export interface IResearchHeaderProps {
 }
 
 const ResearchHeader: React.FunctionComponent<IResearchHeaderProps> = props => {
-  const leftColumnWidth = '6';
-  const leftColumnLabel = '4';
+  const leftColumnWidth = '8';
+  const leftColumnLabel = '3';
   const researchFile = props.researchFile;
 
-  const region = researchFile?.researchProperties
-    ?.map(x => x.property?.region?.description)
-    .filter(x => x !== undefined && x !== '')
+  const regions = researchFile?.researchProperties
+    ?.map(x => x.property?.region)
+    .filter((x): x is Api_TypeCode<number> => !!x && x.description !== '')
```

Comment:
why not just (x:Api_TypeCode<number>)
?

Reply:
Because that tells the transpiler that is the signature of the function. The code above verifies that is a of a particular type at runtime.